app/lib/presenters/complaint.rb

```diff
@@ -0,0 +1,39 @@
+module Presenters
+  class Complaint
+    def initialize(form_builder_payload:)
+      @data = form_builder_payload[:submissionAnswers]
+    end
+
+    def optics_payload
+      {
+        db: 'hmcts',
+        Type: 'Complaint',
+        Format: 'json',
+        RequestMethod: 'Form',
+        RequestDate: @data.fetch(:submissionDate, Date.today),
+        Team: 'INBOX',
+        Customer: {
+          FullName: @data.fetch(:full_name, ''),
+          Address: @data.fetch(:building_street, ''),
+          Town: @data.fetch(:town_city, ''),
+          County: @data.fetch(:county, ''),
+          Postcode: @data.fetch(:postcode, ''),
+          Email: @data.fetch(:email_address, ''),
+          Phone: @data.fetch(:phone, ''),
+          ContactMethod: contact_method
+        },
+        Details: @data.fetch(:complaint_details, ''),
+        Location: @data.fetch(:complaint_location, '')
+      }
+    end
+
+    private
+
+    def contact_method
```

Comment:
you don't need to set this. optics will set this for us

Reply:
Cool, that simplifies this a lot